users/discord/utils/react.js

```diff
@@ -8,7 +8,7 @@ const render = function (element, container) {
   const component = document.createElement(element.tag);
   element.props &&
     Object.keys(element.props).forEach((prop) =>
-      prop == 'onclick' || prop == 'onsubmit'
+      prop == 'onclick' || prop == 'onsubmit' || prop == 'onchange'
```

Comment:
Can we have a comment here that suppose if any other developer want to add any other event listener and they can add it here?